05-customizing-invenio/README.md

```diff
@@ -155,7 +155,7 @@ If we want to be more precise and change a concrete CSS rule we can add it direc
 After changing the file, we have to rebuild our assets using the `invenio
 webpack` command:
 
-```bash
+```console
```

Comment:
Here are you replacing the bash syntax highlighting with console but in other occurences the other way round? (e.g. in [13-securing-your-instance/README.md](https://github.com/inveniosoftware/training/pull/111/files#diff-40659415052de7162a64a8e03acc79f60e6592424c71a8826849917684e1380f)?)

Reply:
`console` can be used when a command has prompt. In this case, `(my-site) $ invenio webpack buildall` contains a prompt. That way, prompt is properly styled when rendered.
`bash` can be used otherwise, as in https://github.com/inveniosoftware/training/pull/111/files#diff-40659415052de7162a64a8e03acc79f60e6592424c71a8826849917684e1380f where command does not contain any prompt.